vitrina/orgs/views.py

```diff
@@ -852,6 +862,914 @@ def form_valid(self, form):
         return redirect(reverse('organization-plans', args=[self.organization.pk]))
 
 
+class OrganizationApiKeysView(
+    LoginRequiredMixin,
+    PermissionRequiredMixin,
+    TemplateView
+):
+    template_name = 'vitrina/orgs/apikeys.html'
+
+    object: Organization
+
+    def dispatch(self, request, *args, **kwargs):
+        self.object = get_object_or_404(Organization, pk=kwargs['pk'])
+        return super().dispatch(request, *args, **kwargs)
+
+    def has_permission(self):
+        return has_perm(
+            self.request.user,
+            Action.MANAGE_KEYS,
+            self.object,
+        )
+
+    def get_context_data(self, **kwargs):
+        context_data = super().get_context_data(**kwargs)
+
+        headers = {
+            "Authorization": CLIENTS_AUTH_BEARER}
```

Comment:
Client tokens expire after 10 days, so we can't save access token in settings. Instead client secret should be stored in settings and access token could be cached for 5 days.

Reply:
So where should the secrets be stored if not in settings.py?